### ISSUE TITLE
Fix accessibility issues for blog

### ISSUE THREAD

**Issue by batpigandme** [tidyverse/tidyverse.org#374]
https://wave.webaim.org/report#/tidyverse.org/blog

Discuss w/ @apreshill what might be best addressed upstream in the theme.

**Comment by batpigandme** [comment]
Notes: 
* Links on list page are "redundant" with the images also linking to the post. Consider using [`aria-hidden`](https://developer.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/) for the thumbnails.
* Code comments are low-contrast, we _could_ provide an [increase contrast control](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G174) (i.e. a ["conforming alternative"](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html#uc-conforming-alt-versions-head)), if we want to preserve that low contrast for other reasons. (Note also that there are definitely ways for people to change syntax highlighting, e.g. see code-highlighting dropdown in the footer of Jim's [plyr-to-dplyr post](https://jimhester.github.io/plyrToDplyr/)).

**Comment by dcossyleon** [comment]
I should be able to target just the `p` tags to bump up body font size but keep all headers the same as they are now

**Comment by dcossyleon** [comment]
@hadley Here's a sample of what the font sizes + headers will look like when the new theme gets ported in. Do you approve?  

https://tidyverse-theme-test.netlify.app/

**Comment by hadley** [comment]
Font size looks good, but maybe text is a bit too "black" now? Maybe try `color: #333` or similar?

**Comment by dcossyleon** [comment]
Oops, the text color shouldn't have changed! Here's take 2, with the original tidyverse off-black: [https://tidyverse-theme-test.netlify.app/](https://tidyverse-theme-test.netlify.app/)

**Comment by hadley** [comment]
Perfect!
